tools/terraform/go.mod

```diff
@@ -0,0 +1,66 @@
+module terraform-provider-fleetdm
```

Comment:
Wondering if we should use a more typical `github.com/fleetdm/fleet/...` module path here? Not totally sure what the implications are, but I found it weird when I saw the unfamiliar `import` syntax for that package. Might be totally ok, just raising in case others have more info on this.

Reply:
Happy to change it if you like

Reply:
I think it's fine, we can always change it later if we see it causes issues.